osm2gtfs/core/stops.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
+
+import sys
+import attr
+
+
+@attr.s
+class Station(object):
+
+    osm_id = attr.ib()
+    tags = attr.ib()
+    lat = attr.ib()
+    lon = attr.ib()
+    name = attr.ib()
+
+    stop_id = attr.ib(default="")
+    osm_type = attr.ib(default="relation")
+    location_type = attr.ib(default=1)
+    osm_url = attr.ib(default="http://osm.org/" +
+                      str(osm_type) + "/" + str(osm_id))
+
+    # Stops forming part of this Station
+    _members = attr.ib(default=attr.Factory(list))
+
+    def __attrs_post_init__(self):
+        '''Populates the object with information obtained from the tags
+
+        '''
+        self.osm_url = "http://osm.org/" + str(
+            self.osm_type) + "/" + str(self.osm_id)
+
+    def set_members(self, members):
+        self._members = members
+
+    def get_members(self):
+        return self._members
+
+    def get_stop_id(self):
+        return self.stop_id
+
+    def set_stop_id(self, stop_id):
+        self.stop_id = stop_id
+
+
+@attr.s
+class Stop(object):
+
+    osm_id = attr.ib()
+    osm_type = attr.ib()
+    tags = attr.ib()
+    lat = attr.ib()
+    lon = attr.ib()
+    name = attr.ib()
+
+    stop_id = attr.ib("")
+    location_type = attr.ib(default=0)
+    osm_url = attr.ib(default=None)
```

Comment:
Why don't we use an abstract `osm-item` class with `osm_id`, `tags`, `name`, `osm_url` and inherit the classes `Station`, `Stop`, `Itinerary` and Line`? Or eventually another parent grouping for the first and latter two? Is this possible in combination with `attrs`?

Reply:
This is a good idea. We should check.

Reply:
I've seen that the former classes already had inheritance from `BaseRoute`, at least.

Reply:
ok, thanks for the suggestions. Based on this, I did:

* Made all existing data elements `Line`, `Itinerary`, `Station` and `Stop` inheriting the basic attributes from a new class `Element`.
* Moved add data elements into `elements.py`